integration-test/src/main/java/com/scalar/db/api/DistributedTransactionIntegrationTestBase.java

```diff
@@ -93,10 +111,32 @@ protected String getNamespaceBaseName() {
   }
 
   private void createTables() throws ExecutionException {
+    TableMetadata.Builder tableMetadata =
```

Comment:
Up until now `DistributedTransactionIntegrationTestBase` was performing tests on a table composed of columns of INT type only.
I updated only the test that targeted the most generic use case for each operation to target all data types.